package/boot/rkbin/Makefile

```
@@ -25,13 +25,20 @@ define Trusted-Firmware-A/Default
   BUILD_TARGET:=rockchip
 endef
 
+define Trusted-Firmware-A/rk3566
+  BUILD_SUBTARGET:=armv8
+  ATF:=rk35/rk3568_bl31_v1.43.elf
+  TPL:=rk35/rk3566_ddr_1056MHz_v1.18.bin
+endef
+
 define Trusted-Firmware-A/rk3568
   BUILD_SUBTARGET:=armv8
   ATF:=rk35/rk3568_bl31_v1.43.elf
   TPL:=rk35/rk3568_ddr_1560MHz_v1.18.bin
 endef
 
 TFA_TARGETS:= \
+    rk3566 \
```

```suggestion
	rk3566 \
```
use `<tab>` insead of 4 `<space>`

---

fixed